school/student.rb

```diff
@@ -0,0 +1,12 @@
+require './school/person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] The parameters for the def initialize should take the same one as the person parameters, not the classroom.